client/src/components/HomePage.tsx

```diff
@@ -11,13 +16,138 @@ const MainHeader = styled.h1`
   width: 100%;
 `;
 
-export type UserIdProps = {
-  userId: number;
-  setUserId: (userId: number) => void;
-};
+const SecondHeader = styled.h1`
+  font-size: 4rem;
+  color: ${CONTENT_COLOR};
+  text-align: center;
+`;
+
+const Content = styled.h2`
+  font-size: 3rem;
+  color: ${PRIMARY_COLOR};
+  text-align: center;
+`;
+
+const ContentText = styled.h3`
+  color: ${CONTENT_COLOR};
+  text-align: center;
+`;
+
+const ContentBlurb = styled.section`
+  border-bottom: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-top: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-right: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-left: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
```

Comment:
two things:
- take out the space between the color and the opaque. the opacity won't do anything like this
- when all 4 `border-*` are the same, you could use this shorthand: `border: 1px solid color`